alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java

```diff
@@ -31,7 +31,7 @@ private void registerCoreStats() {
                 Thread.currentThread().sleep(10_000);
                 trackerRegistry = coreAdminHandler.getTrackerRegistry();
             } catch (InterruptedException e) {
-                e.printStackTrace();
+                logger.error(e.getMessage());
```

Comment:
use `logger.error("Short text describing what failed", e)`

Reply:
Hmm, can we leave this for latter? 
There are other parts of the code which need some cleaning and I planned to use the released artifact today.

Reply:
Ignore, I'll do it.

Reply:
You can replace all occurrences with:  `logger.error(e.getMessage(), e)`

Reply:
That should do it.

Reply:
No problem, I added explicit messages.